File: src/main/java/com/p2p/lending/service/ClapplyforService.java
package com.p2p.lending.service;

import java.util.List;
import java.util.Map;

import com.p2p.lending.entity.Clapplyfor;

public interface ClapplyforService {
	public List<Clapplyfor> queryClapplyfors(Map<String, Object> map);
	
	public int insertClapplyfor(Map<String, Object> map);
	
	public int updateClapplyforState(Map<String, Object> map);
}

File: src/main/java/com/p2p/lending/service/ClapplyforService.kt
package com.p2p.lending.service

import com.p2p.lending.entity.Clapplyfor
import org.springframework.stereotype.Service

@Service
interface ClapplyforService {
    fun queryClapplyfors(map: Map<String, Any?>): List<Clapplyfor>

    fun insertClapplyfor(map: Map<String, Any?>): Int

    fun updateClapplyforState(map: Map<String, Any?>): Int
}
